crates/editor/src/git/blame.rs

```diff
@@ -253,10 +253,10 @@ impl GitBlame {
     fn generate(&mut self, cx: &mut ModelContext<Self>) {
         let buffer_edits = self.buffer.update(cx, |buffer, _| buffer.subscribe());
         let snapshot = self.buffer.read(cx).snapshot();
-        let blame = self.project.read(cx).blame_buffer(&self.buffer, None, cx);
+        let blame = Project::blame_buffer(&self.project, &self.buffer, None, cx);
```

Comment:
I'm not sure I'm following — why do we need this change?

Reply:
I initially needed it to get access to the Model and ModelContext objects but now I don't. I'll revert that part and push another commit